//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcdeg_T_A.h
//
// Code generation for function 'abcdeg_T_A'
//

#ifndef ABCDEG_T_A_H
#define ABCDEG_T_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdeg_T_A(double V_init, double A_init, double A_wayp, double V_max,
                double A_max, double J_max, double J_min, double T,
                double t[14]);

#endif
// End of code generation (abcdeg_T_A.h)
